src/main/java/dk/kb/present/View.java

```diff
@@ -85,30 +88,55 @@ public MediaType getMime() {
     }
 
     @Override
-    public String apply(String recordID, String content, String child) {
-        final Map<String, String> metadata = new HashMap<>();
-        metadata.put("recordID", recordID);
-        metadata.put("origin", origin);
+    public String apply(DsRecordDto record) {
+        final Map<String, String> metadata = createBasicMetadataMap(record);
 
-        if (child != null ) {
-            metadata.put("childRecord", child);
-        }
+        String content = record.getData();
+
+        switch (strategy) {
+            case "manifestation":
```

Comment:
There are no strategies defined in the `ds-present-kb-collections.yaml`. Maybe it could be injected there for a few `View`s and with a comment explaining what it is about, in order to make it easier to create other setups?